src/adapters/rhythmone.js

```diff
@@ -224,53 +186,76 @@ module.exports = function(bidManager, global, loader){
     return endpoint;
   }
   
+  function sendAuditBeacon(placementId){
+    var data = {
+        doc_version : 1,
+        doc_type : "Prebid Audit",
+        placement_id: placementId
+      },
+      ao = document.location.ancestorOrigins,
```

Comment:
It's not a concern, it is only available in webkit browsers TMK so it will only be used if there.